### ISSUE TITLE
Broken import in production build

### ISSUE THREAD

**Issue by balinterdi** [ef4/ember-auto-import#5]
I did the following:

```
ember install ember-auto-import
yarn add --dev lodash-es
```

And then imported a lodash function as follows:

```js
import { difference } from 'lodash';
```

It worked like a charm in development but when I deployed the new version, I got an import error in production:

> Could not find module `lodash` imported from `<module-name>`

I couldn't import the `lodash` module from the console manually, either.

I also checked vendor.js and didn't seem to find any trace of `lodash`.

Our deployment creates the production build (`ember build -e production`) and then uses a [Netlify command](https://www.netlify.com/) to push it to their servers.

**Comment by ef4** [comment]
Hypothesis: you are importing from `lodash` instead of `lodash-es`. In your development environment you got lucky and had another copy of `lodash` that was resolvable, but there wasn't one wherever your production build ran.

**Comment by ef4** [comment]
It would be nice to make ember-auto-imports error out if you try to import something that isn't in your `devDependencies`, but that would require us to be aware of all the other possible sources of modules.

**Comment by balinterdi** [comment]
That's definitely possible, great hypothesis.

I think I originally tried to import from `lodash-es` and got an error – that's why I switched to importing `lodash` but I'm not totally certain.

I'll not venture to possibly break the "real" app in production again but I'll try it on the R&R app this week and then report back.

Thank you!

**Comment by balinterdi** [comment]
I tried it again, this time with the R&R app but it doesn't work.

I did

    $ ember install ember-auto-import
    $ yarn add --dev lodash-es

and then used `capitalize` in a helper:

```js
import { helper } from '@ember/component/helper';
import { capitalize } from 'lodash-es';

export function cap(input) {
  let words = input.toString().split(/\s+/).map((word) => {
    return capitalize(word.charAt(0)) + word.slice(1);
  });
  return words.join(' ');
}

export default helper(cap);
```

When the app boots up, here's the exact error I get:

```
Uncaught Error: Could not find module `lodash-es` imported from `rarwe/helpers/capitalize`
    at missingModule (loader.js:247)
    at findModule (loader.js:258)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at Module.findDeps (loader.js:168)
    at findModule (loader.js:262)
    at requireModule (loader.js:24)
    at Class._extractDefaultExport (index.js:411)
    at Class.resolveOther (index.js:114)
    at Class.superWrapper [as resolveOther] (ember-utils.js:428)
```

Changing the import to `import { capitalize } from 'lodash-es';` works but, as you point out above, it's incidental. `lodash` is a dependency of several npm packages that the app uses and will work in development but break the app in production.

**Comment by ef4** [comment]
I'm pretty sure this is fixed in 1.0.0-alpha.0.

**Comment by balinterdi** [comment]
Thank you, Ed, I just verified on the RaRwE app and this indeed works now.
